scripts/src/services.ts

```diff
@@ -0,0 +1,99 @@
+import * as moment from "moment";
+import { readFileSync } from "fs";
+import * as jsonwebtoken from "jsonwebtoken";
+import { Request, Response, RequestHandler } from "express";
+
+import { randomItem, path } from "./utils";
+import { getConfig } from "./config";
+
+class KeyMap extends Map<string, { algorithm: string; key: Buffer; }> {
+	public random() {
+		const entries = Array.from(this.entries()).map(([id, key]) => ({
+			id,
+			key: key.key,
+			algorithm: key.algorithm
+		}));
+
+		return randomItem(entries);
+	}
+}
+
+const CONFIG = getConfig();
+const KEYS = new KeyMap();
```

Comment:
move these to the top of the file (right after all the imports of course)

Reply:
This: `const KEYS = new KeyMap();` has to come up after the class definition of `KeyMap`, otherwise it will be an error (compilation and runtime).

As it has to be there, I also thought that it would be good to have the other const (`CONFIG`) there as well